tasks/main.yml

```diff
@@ -150,6 +169,22 @@
     - postfix
     - postfix-generic-table
 
+- name: configure header checks
+  template:
+    src: etc/postfix/header_checks.j2
+    dest: /etc/postfix/header_checks
+    owner: root
+    group: root
+    mode: 0644
+  when: postfix_header_checks is defined
+  with_items: "{{ postfix_header_checks }}"
```

Comment:
Is the `with_items` needed?

Reply:
no it's not needed. removed in newest commit